python/lsst/sims/maf/plots/spatialPlotters.py

```diff
@@ -639,3 +640,93 @@ def __call__(self, metricValueIn, slicer, userPlotDict, fignum=None, ):
             cb1.solids.set_edgecolor("face")
         fig = plt.gcf()
         return fig.number
+
+
+class LambertSkyMap(BasePlotter):
+    """
+    Use basemap and contour to make a Lambertian projection.
+    Note that the plotDict can include a 'basemap' key with a dictionary of
+    kwargs to use with the call to Basemap.
+    """
+
+    def __init__(self):
+        self.plotType = 'SkyMap'
+        self.objectPlotter = False
+        self.defaultPlotDict = {'basemap':{'projection':'nplaea', 'boundinglat':20,
+                                           'lon_0':0., 'resolution':'l', 'celestial':True},
+                                'cbar':True, 'cmap':plt.cm.jet, 'levels':200,
+                                'cbarFormat':'%.2f','cbar_edge':True, 'zp':None,
+                                'normVal':None, 'percentileClip':False, 'colorMin':None,
+                                'colorMax':None, 'linewidths':0}
+
+    def __call__(self, metricValue, slicer, userPlotDict, fignum=None):
+
+        plotDict = {}
+        plotDict.update(self.defaultPlotDict)
+        plotDict.update(userPlotDict)
+
+        if plotDict['zp'] is not None :
+            metricValue = metricValue - plotDict['zp']
+        if plotDict['normVal'] is not None:
+            metricValue = metricValue/plotDict['normVal']
+
```

Comment:
hey - I forget if we decided this was safe or not (for the metricValues). have you confirmed that if you pass an array, then alter it in the run method, that it is not changed outside of the run method here?  For a lot of things, this is true, but I wonder about complex numpy arrays, whether they're always passed as a copy or as a view on the original.